libcontainer/message_linux.go

```diff
@@ -56,5 +56,5 @@ func (msg *Bytemsg) Serialize() []byte {
 }
 
 func (msg *Bytemsg) Len() int {
-	return syscall.NLA_HDRLEN + len(msg.Value) + 1 // null-terminated
+	return NLA_HDRLEN() + len(msg.Value) + 1 // null-terminated
```

Comment:
yes i meant adding variables in the two files instead of using a function :p sorry for making it not clear.